src/views/Posters/index.jsx

```diff
@@ -0,0 +1,553 @@
+import { useState, useEffect } from 'react';
+import {
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  Grid,
+  Typography,
+  CardMedia,
+  CardActionArea,
+  Dialog,
+  DialogContent,
+} from '@mui/material';
+import UploadForm from './Forms/Forms/UploadForm';
+import GordonSnackbar from 'components/Snackbar';
+import { Participation } from 'services/membership';
+import membershipService from 'services/membership';
+import involvementService from 'services/involvements';
+import sessionService from 'services/session';
+import { useNavigate } from 'react-router-dom';
+import { getCurrentPosters, hidePoster } from 'services/poster';
+import { useLocation } from 'react-router-dom';
+import CropPoster from './Forms/Forms/CropPoster';
+import { AuthGroup } from 'services/auth';
+import { useAuthGroups, useNetworkStatus, useUser } from 'hooks';
+import FileUploadedRoundIcon from '@mui/icons-material/FileUploadRounded';
+import GordonLoader from 'components/Loader';
+const LOADER_SIZE = 50;
+
+const Posters = () => {
+  const [openUploadForm, setOpenUploadForm] = useState(false);
+  const [loadingInvolvements, setLoadingInvolvements] = useState(true);
+  const { profile } = useUser();
+  const [myUploadedPosterIds, setMyUploadedPosterIds] = useState(new Set());
+  const [allInvolvements, setAllInvolvements] = useState([]);
+  const [myInvolvements, setMyInvolvements] = useState([]);
+  const [selectedSession, setSelectedSession] = useState('');
+  const [croppedImage, setCroppedImage] = useState(null);
+  const location = useLocation();
+  const [openCropPoster, setOpenCropPoster] = useState(false);
+  const [allPosters, setAllPosters] = useState([]);
+  const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
+  const [posterToDelete, setPosterToDelete] = useState(null);
+  const isSiteAdmin = useAuthGroups(AuthGroup.SiteAdmin);
+  const [openEditDialog, setOpenEditDialog] = useState(false);
+  const [posterToEdit, setPosterToEdit] = useState(null);
+
+  const [snackbar, setSnackbar] = useState({
+    message: '',
+    severity: '',
+    open: false,
+  });
+
+  const createSnackbar = (message, severity, link, linkText) => {
+    setSnackbar({
+      message,
+      severity,
+      open: true,
+      link,
+      linkText,
+    });
+  };
+
+  const isOnline = useNetworkStatus();
+  const navigate = useNavigate();
+
+  const isMyClub = (ClubCode, posterID) => {
+    return (
+      myInvolvements.some(
+        (inv) =>
+          inv.ActivityCode === ClubCode &&
+          ['MEMBR', 'LEAD', 'ADV', 'GUEST'].includes(inv.Participation),
+      ) || myUploadedPosterIds.has(posterID)
+    );
+  };
+
+  const pizzaSlice = allPosters.filter((item) => isMyClub(item.ClubCode, item.ID));
```

Comment:
Why is this called pizzaSlice?  would something like myClubPosters make more sense?